modules/logging-release-notes-6-4-0.adoc

```diff
@@ -0,0 +1,113 @@
+
+* Before this update, the `ClusterLogForwarder` API required the URL for OTLP endpoints to terminate with `v1/logs`. With this update, this requirement has been relaxed to allow any URL that specifies an `http` or `https` protocol. (link:https://issues.redhat.com/browse/LOG-7582[LOG-7582])
+
+*  Before this update, any request that exceeded a Kafka broker's `message.max.size` value would be rejected because the collector's tuning did not correctly set an allowable producer configuration.
+With this update, you can set the collector's kafka client configuration to allow message sizes that are equal to or smaller than the `MaxSize` value. (link:https://issues.redhat.com/browse/LOG-7608[LOG-7608])
+
+* Before this update, the prune filter failed to remove the `.openshift.sequence` field from the log record. With this update, the field is correctly pruned from the log record. (link:https://issues.redhat.com/browse/LOG-7620[LOG-7620])
+
+* Before this update, the prune filter failed to remove the `.kubernetes.container_iostream` field from the log record. With this fix, the field is now correctly pruned from the log record. (link:https://issues.redhat.com/browse/LOG-7622[LOG-7622])
+
+
+[id="logging-release-notes-6-4-0-deprecation-notice_{context}"]
+== Deprecation notice
+
+In this release, the 'observability.openshift.io/max-unavailable-rollout' annotation is deprecated will be removed in a future release.
+The annotation has been replaced by the `spec.collector.maxUnavailable` field in the `ClusterLogForwarder` resource.
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_logging/6.4/html/configuring_logging/cluster-logging-collector#configuring-pod-rollout-strategy_cluster-logging-collector[Configuring pod rollout strategy].
+
+
+[id="logging-release-notes-6-4-0-removal-notice_{context}"]
+== Removal notice
+
+In this release, the `observability.openshift.io/use-apiserver-cache` annotation has been removed.
+With this release, kube-api caching is now always enabled.
+For more information, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_logging/6.4/html/configuring_logging/cluster-logging-collector#configuring-pod-rollout-strategy_cluster-logging-collector[Configuring pod rollout strategy].
+
+
+[id="logging-release-notes-6-4-0-known-issues_{context}"]
+== Known issues
+////
+
+TODO: Check if this is still valid
+
+
+* When you forward logs to a syslog output, the produced message format is inconsistent between Fluentd and Vector log collectors. Vector messages are within quotation marks; Fluentd messages are not. As a consequence, users might experience issues with their tool integrations when they migrate from Fluentd to Vector. (link:https://issues.redhat.com/browse/LOG-7007[LOG-7007])
+
+////
+
+* When network policies are enabled in {loki-op} and an in-cluster object storage is used, the network policies do not allow access to the object storage.
```

Comment:
Are s3 and swift "in-cluster object storage"?  I think there are external object storage which are out of cluster? @xperimental

Reply:
We talked about this in the status meeting and decided to reword the entry.